### PR TITLE
PEP 8002: Add information about Project Jupyter governance

### PR DESCRIPTION
**Pull request by willingc**

### CODE REVIEW

File: pep-8002.rst [hunk 1]
```diff
@@ -371,6 +372,114 @@ code they deliver allows different teams to have different
...
+Steering Council may call for a vote to ratify the PR changes. Acceptance
+requires a minimum of 80% of the Steering Council to vote and at least 2/3 of
+the vote must be positive. The BDFL can act alone to accept or reject changes
+or override the Steering Council decision; though this would be an extremely
+rare event.
+
+Key people and their functions
+------------------------------
+
+The key people in Jupyter's Governance are the BDFL, Fernando Perez, and the
```
Comment (pitrou) [inline pep-8002.rst]: I would expect a sentence explaining whether the BDFL was there from the start (is he the project founder?) and whether it's truly "for life".

File: pep-8002.rst [hunk 2]
```diff
@@ -371,6 +372,114 @@ code they deliver allows different teams to have different
...
+Steering Council. Contributors can be given a special status of core contributor.
+Some may also be Institutional Contributors, who are individuals who contribute
+to the project as part of their official duties at an Institutional Partner.
+
+Core contributors are individuals who are given rights, such as commit privileges,
+to act in the best interest of the project within their area of expertise or
+subproject. An existing core contributor typically recommends someone be given
+core contributor rights by gathering consensus from project leads.
```
Comment (pitrou) [inline pep-8002.rst]: Who are the project leads? Existing core contributors, or the Steering Council?

File: pep-8002.rst [hunk 3]
```diff
@@ -371,6 +372,114 @@ code they deliver allows different teams to have different
...
+------------------------
+
+Project Jupyter is made up of a number of GitHub organizations and subprojects
+within those organizations. Primary work happens via GitHub issues and pull
+requests. Approving a pull request by any team member allows it to be merged
+without further process. All merged pull requests end up in the next stable
+release of a subproject.
+
+There is a weekly, public Project-wide meeting that is recorded and posted on
+YouTube. Some larger GitHub organizations, e.g. JupyterLab and JupyterHub, may
```
Comment (pitrou) [inline pep-8002.rst]: Which, I presume, are Jupyter sub-projects or sub-teams?

**Comment by willingc** [comment]
Thanks @pitrou for the comments and review. I've added additional clarification on the items that you suggested.

**Pull request merged**